src/prngs/chacha.c

```diff
@@ -0,0 +1,213 @@
+/* LibTomCrypt, modular cryptographic library -- Tom St Denis
```

Comment:
Did you follow some guide "how to use ChaCha as a prng" or is this all your work?

From my point of view it looks very good, but if you created this I'd like to have someone else with crypto expertise to review this code.